tests/AutoReview/ProjectCodeTest.php

```diff
@@ -416,6 +413,62 @@ static function (\ReflectionMethod $reflectionMethod) use ($reflectionClass) {
         }
     }
 
+    /**
+     * @dataProvider provideSrcClassCases
+     * @dataProvider provideTestClassCases
+     *
+     * @param string $className
+     */
+    public function testAllCodeContainSingleClassy($className)
+    {
+        $headerTypes = [
+            T_ABSTRACT,
+            T_AS,
+            T_COMMENT,
+            T_DECLARE,
+            T_DOC_COMMENT,
+            T_FINAL,
+            T_LNUMBER,
+            T_NAMESPACE,
+            T_NS_SEPARATOR,
+            T_OPEN_TAG,
+            T_STRING,
+            T_STRING,
```

Comment:
I think we don't need double-check T_STRING 😄

Reply:
: D